doc/release/1.4.0-notes.rst

```diff
@@ -69,18 +184,121 @@ function supports specifying dimensions along which to do the convolution.
 
 `scipy.signal.cwt` now supports complex wavelets.
 
+The implementation of ``choose_conv_method`` has been updated to reflect the 
+new FFT implementation. In addition, the performance has been significantly 
+improved (with rather drastic improvements in edge cases).
+
+The function ``upfirdn`` now has a ``mode`` keyword argument that can be used
+to select the signal extension mode used at the signal boundaries. These modes
+are also available for use in ``resample_poly`` via a newly added ``padtype``
+argument.
+
+`scipy.signal.sosfilt` now benefits from Cython code for improved performance
+
+`scipy.signal.resample` should be more efficient by leveraging ``rfft`` when
+possible
+
 `scipy.sparse` improvements
 ---------------------------
+It is now possible to use the LOBPCG method in `scipy.sparse.linalg.svds`.
+
```

Comment:
Can you copy the `pydata/sparse` entry to here as well?

Reply:
it is already there, a few lines down